src/viper/harness.yml

```diff
@@ -70,6 +70,10 @@ attributes:
         enabled: true
         hosts:
           - = 'gateway-' ~ @('hostname')
+        tls:
+          - hosts:
+            - = 'gateway-' ~ @('hostname')
+            secretName: viper-gateway-letsencrypt-http-cert
```

Comment:
as these are specified, they are no longer optional. Clusters that use a wildcard certificate I assume shouldn't have this